<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choosePictureClass</class>
 <widget class="QMainWindow" name="choosePictureClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>choosePicture</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(240, 240, 240);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="3,0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(198, 200, 191);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="11,1,5">
          <item>
           <widget class="QGraphicsView" name="img_GraphicsView">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(198, 200, 191);
font: 12pt &quot;黑体&quot;;</string>
            </property>
            <property name="text">
             <string>日志文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="Log_TextEdit">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QWidget" name="widget_4" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(198, 200, 191);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0" columnstretch="1,1,1">
          <item row="2" column="0" colspan="3">
           <widget class="QWidget" name="widget_9" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="1">
              <widget class="QCheckBox" name="Log_checkBox">
               <property name="text">
                <string>Log文件</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QCheckBox" name="Config_checkBox">
               <property name="text">
                <string>配置文件</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_10">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;黑体&quot;;</string>
               </property>
               <property name="text">
                <string>Log/配置文件</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QWidget" name="widget_7" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0,0,0,0,0,0">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;黑体&quot;;</string>
               </property>
               <property name="text">
                <string>FOB卡</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="AllFob_checkBox">
               <property name="text">
                <string>全选</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Yang_checkBox">
               <property name="text">
                <string>阳性</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="RouYang_checkBox">
               <property name="text">
                <string>弱阳性</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Yin_checkBox">
               <property name="text">
                <string>阴性</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="AllcheckBox">
               <property name="text">
                <string>复选</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Preferred</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>123</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QWidget" name="widget_5" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;黑体&quot;;</string>
               </property>
               <property name="text">
                <string>细胞</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="AllCell_checkBox">
               <property name="text">
                <string>全选</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="RedCell_checkBox">
               <property name="text">
                <string>红细胞</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="WhiteCell_checkBox">
               <property name="text">
                <string>白细胞</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Mold_checkBox">
               <property name="text">
                <string>霉菌</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Preferred</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>117</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QWidget" name="widget_6" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;黑体&quot;;</string>
               </property>
               <property name="text">
                <string>颜色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="AllColor_checkBox">
               <property name="text">
                <string>全选</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Yellow_checkBox">
               <property name="text">
                <string>黄色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Red_checkBox">
               <property name="text">
                <string>红色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Brown_checkBox">
               <property name="text">
                <string>褐色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Black_checkBox">
               <property name="text">
                <string>黑色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;黑体&quot;;</string>
               </property>
               <property name="text">
                <string>性状</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="AllTraits_checkBox">
               <property name="text">
                <string>全选</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Rash_checkBox">
               <property name="text">
                <string>稀便</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Soft_checkBox">
               <property name="text">
                <string>软便</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(198, 200, 191);
background-color: rgb(240, 240, 240);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="0">
              <widget class="QPushButton" name="leftTurn">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);
font: 14pt &quot;方正姚体&quot;;
font: 14pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>上一张</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QPushButton" name="RightUid">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);
font: 14pt &quot;方正姚体&quot;;
font: 14pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>下个样本</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="LeftUid">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);
font: 14pt &quot;方正姚体&quot;;
font: 14pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>上个样本</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="RightTurn">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);
font: 14pt &quot;方正姚体&quot;;
font: 14pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>下一张</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QPushButton" name="Find_Button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);
font: 14pt &quot;方正姚体&quot;;
font: 14pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>查询</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2" colspan="2">
              <widget class="QPushButton" name="Save_Pic">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);
font: 14pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>导出图片</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QWidget" name="widget_8" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="3" column="3">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>结束时间</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>起始时间</string>
               </property>
              </widget>
             </item>
             <item row="3" column="5">
              <widget class="QDateEdit" name="End_dateEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="dateTime">
                <datetime>
                 <hour>0</hour>
                 <minute>0</minute>
                 <second>0</second>
                 <year>2018</year>
                 <month>1</month>
                 <day>1</day>
                </datetime>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDateEdit" name="Start_dateEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="readOnly">
                <bool>false</bool>
               </property>
               <property name="dateTime">
                <datetime>
                 <hour>0</hour>
                 <minute>0</minute>
                 <second>0</second>
                 <year>2018</year>
                 <month>1</month>
                 <day>1</day>
                </datetime>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>图片个数</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>UID</string>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QLineEdit" name="UID_lineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="Count_Pic">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>IBeamCursor</cursorShape>
               </property>
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>0/0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>进度条</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="choosepicture.qrc"/>
 </resources>
 <connections/>
</ui>
